src/Bigtable/Connection/TableAdminConnectionInterface.php

```diff
@@ -0,0 +1,254 @@
+<?php
+/*
+ * Copyright 2018, Google LLC All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Cloud\Bigtable;
+
+use Google\Cloud\Bigtable\ConnectionInterface;
+
+/**
+ * Connection to Bigtable TableAdmin API
+ */
+interface TableAdminConnectionInterface extends ConnectionInterface
+{
+
+    /**
+     * Creates a new table in the specified instance.
+     *
+     * @param string $tableId      The name by which the new table should be referred to within the parent
+     *                             instance, e.g., `foobar` rather than `<parent>/tables/foobar`.
+     *
+     * @param array  $optionalArgs {
```

Comment:
We need parameters for multiple column families.  Each Column Family should have garbage collection information.

Here's the definition of a [`Table`](https://github.com/googleapis/googleapis/blob/master/google/bigtable/admin/v2/table.proto#L31).  It has a map of [`ColumnFamily`](https://github.com/googleapis/googleapis/blob/master/google/bigtable/admin/v2/table.proto#L143) where the map's key is the name of the family, and the value is a `ColumnFamily`.  A `ColumnFamily` has a [`GCRule`](https://github.com/googleapis/googleapis/blob/master/google/bigtable/admin/v2/table.proto#L154) which is a complex object that defines how Bigtable will delete old data.

It would be great to see a Table object in this PR, and how that fits into the API.